### PR TITLE
Logged axis

### PR DESCRIPTION
**Pull request by jordansread**
addressing #71

### CODE REVIEW

**Comment by jordansread** [comment]
@jiwalker-usgs and @ldecicco-USGS could use some advice on this

File: R/calc_views.R [hunk 1]
```diff
@@ -88,119 +86,83 @@ group_views <- function(gsplot){
...
-        client_lims <- lims_from_client(side_components, var='ylim', side)
-        lims[!is.na(client_lims)] <- client_lims[!is.na(client_lims)]
-      } else {
-        dataViews <- views[!(names(views) %in% "legend")]
-        dataViews <- do.call(c, unname(dataViews))
-        dataViews <- dataViews[!names(dataViews) %in% c("axis","gs.config")]
-        dataViews <- do.call(c, unname(dataViews))
-        yData <- do.call(c, unname(dataViews[names(dataViews) %in% c('y','y1','y0')]))
+set_view_lab <- function(views){
+  views <- set_view_list(views, var = 'ylab', na.action="")
```
Comment (jiwalker-usgs) [inline R/calc_views.R:115]: should this be "xlab"?

Comment (jordansread) [inline R/calc_views.R:115]: isn't xlab on the line below it?

Comment (jiwalker-usgs) [inline R/calc_views.R:115]: nope, that one is ylab as well

**Comment by jiwalker-usgs** [comment]
Other than that, I may need to talk to know what the problem is (a bit out of touch with latest stuff here).

File: R/calc_views.R (hunk 1 above)
Comment (jordansread) [inline R/calc_views.R:115]: Old diff?

**Comment by jordansread** [comment]
@jiwalker-usgs would be good to talk tomorrow if you have ~15 mins or so.

**Comment by mhines-usgs** [comment]
@jread-usgs  I can get the examples to run after pulling logged_axis down, but I am not really understanding the problem you're envisioning with the no data/y/xlims.

**Comment by ldecicco-USGS** [comment]
Oh...and now I'm seeing the whole conversation (the previous comment was about a single commit)

**Comment by ldecicco-USGS** [comment]
Reading all these comments makes me so sad...I was so proud of that code...now poof...gone

**Pull request merged**